### ISSUE TITLE
Prisma studio delete button for record

### ISSUE THREAD

**Issue by sandbox-apps** [prisma/studio#739]
Integrate delete button for prisma studio

**Comment by madebysid** [comment]
Could you clarify? You should be able to select a record using the checkboxed on the left and delete them, right?

**Comment by sandbox-apps** [comment]
Yes, my fault sorry. Im thinking of a literal delete along side with the row. Maybe a delete button per row or we can right click the row and it will show a delete this row menu. Instead of going to check the sidebar and clicking the delete at the top.

<img width="927" alt="Screen Shot 2021-07-15 at 1 20 40 AM" src="https://user-images.githubusercontent.com/86398599/125665110-e08ae2ad-d965-40ae-985b-26b6c4122d4d.png">

**Comment by igalklebanov** [comment]
Hey :wave:

Prisma ORM 7 introduces a brand new standalone [Studio](https://www.prisma.io/docs/orm/tools/prisma-studio), built from the ground up to work everywhere, with or without Prisma ORM. Consequently, all "old Studio" issues are being closed as there won't be any further work on it. :mega:

The new Studio is also available in the Prisma VS Code extension and in the Prisma Console application. :electric_plug:

Help shape, improve and fix the new Studio with us in this repository by submitting bug reports and feature requests. :construction_worker:
